modules/cnf-understanding-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+[id="cnf-understanding-topology-aware-lifecycle-operator-config_{context}"]
+= Understanding the Topology Aware Lifecycle Operator configuration
+
+The Topology Aware Lifecycle Operator manages the deployment of upgrade policies to one or more clusters in a given period of time, which is 4 hours by default. The Topology Aware Lifecycle Operator uses the `ClusterGroupUpgrade` CR. The `ClusterGroupUpgrade` CR can be in the following states:
+
+`UpgradeNotStarted`:: In this state, the `ClusterGroupUpgrade` CR is created and the `remediationAction` field is set to `inform`.
++
+The operator builds a remediation plan based on the clusters list and the `remediationStrategy` fields. The operator transitions to the `UpgradeNotCompleted` state once the `remediationAction` field is set to `enforce`.
++
```

Comment:
Should the sample CR here reflect 4.10 as release is 4.10, or maybe we can employ variable substitution